content/cdi-i/exercicios/0001-fichas-2020-2021.md

```diff
@@ -0,0 +1,135 @@
+  - [Resolução Prof](https://bit.ly/3HH46ar)
+
+  - [Enunciado Exs Extra](https://bit.ly/30NmEom)
+  - [Resolução Exs Extra](https://drive.google.com/file/d/1eezyU-NCRcPahZQmZ7FZZFjk6bz9c6Mb/view)
+
+- Semana 6
+
+  - [Enunciado](https://bit.ly/3HLYxaP)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/1CuMfNGkzE7CUnCUB5JG6CqEv3tK-pQ9d/view)
+  - [Resolução Prof](https://bit.ly/3FB9rhA)
+
+  - [Enunciado Exs Extra](https://bit.ly/3xct3Wb)
+  - [Resolução Exs Extra](https://drive.google.com/file/d/14bPsG5PrthfQFOahaJhjZaaQMC36j5Gt/view)
+
+- Semana 7
+
+  - [Enunciado](https://bit.ly/3oSu3v6)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/1Lq9wE5HanI8GEgjm9AC3QFGtrMjOCyk3/view)
+  - [Resolução Prof](https://bit.ly/3xa8E4l)
+
+  - [Enunciado Exs Extra](https://bit.ly/3nEcEGW)
+  - [Resolução Exs Extra](https://drive.google.com/file/d/1RW5W5a6js7YgRC8QmmpTN26krYTQru9U/view)
+
+- Semana 8
+
+  - [Enunciado](https://bit.ly/3cAwTPF)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/1eYOtO4NYuIP9ktec1VhrHTTW5QISZY06/view)
+  - [Resolução Prof](https://bit.ly/2ZcXM97)
+
+  - [Enunciado Exs Extra](https://bit.ly/3nHVusg)
+  - [Resolução Exs Extra](https://drive.google.com/file/d/1hCaNaiR3mIW6xJiIUYx3bt9D8RoH-2Dy/view)
+
+- Semana 9
+
+  - [Enunciado](https://bit.ly/3nDpXHN)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/19I2rtcWzkgc0Dly9S4HbGDh76z-MT0Tn/view)
+  - [Resolução Prof](https://bit.ly/32oHP0V)
+
+  - [Enunciado Exs Extra](https://bit.ly/30KZLSw)
+  - [Resolução Exs Extra](https://drive.google.com/file/u/1/d/1XIkp-ARCKWtlepBcyFGV1-OveJ1oQtRL/view?usp=sharing)
+
+- Semana 10
+
+  - [Enunciado](https://bit.ly/3l1DAPd)
+  - [Resolução Diogo Correia](https://drive.google.com/file/u/1/d/1f1oWNXxXNLQJKs3k8sD2NlcH6SzTgq2_/view?usp=sharing)
+  - [Resolução Prof](https://bit.ly/3cBVrrl)
+
+  - [Enunciado Exs Extra](https://bit.ly/3kVBEYn)
+  - [Resolução Exs Extra](https://drive.google.com/file/u/1/d/1C7H92qbnB7cg_Jz9gz47bPoSIYXy3pRi/view)
+
+- Semana 11
+
+  - [Enunciado](https://bit.ly/3xe1ewF)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/1BbPHJafxjhCAUtZsgJmjATvHTs2WfPzq/view)
+  - [Resolução Prof](https://bit.ly/3CIXrZz)
+
+  - [Enunciado Exs Extra](https://bit.ly/3DJho3E)
+  - [Resolução Exs Extra](https://drive.google.com/file/u/1/d/1G3EA28MMFLc2-OjDluxUuqKu77tN5SAc/view)
+
+- Semana 12
+
+  - [Enunciado](https://bit.ly/3CHxtFC)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/1wgMIM8-md398pcsnATqo1x-zkNulk4Mw/view)
+  - [Resolução Prof](https://bit.ly/3nCruOi)
+
+  - [Enunciado Exs Extra](https://bit.ly/30T5xlx)
+  - [Resolução Exs Extra](https://drive.google.com/file/d/1Kbr_CSrut8W2OEKqyy-Ir4TX__POUEqo/view)
+
+- Semana 13
+
+  - [Enunciado](https://bit.ly/3DGywqT)
+  - [Resolução Diogo Correia](https://drive.google.com/file/d/1nhqVTC2FwoTWXf-EIrP2nhDUbD2EOgvf/view)
```

Comment:
It'd be nice if all the files were hosted on Google Drive instead of going through bit.ly and notion